### PR TITLE
Remove reference to live-1

### PR DESCRIPTION
**Pull request by sablumiah**
Remove reference to live-1 in runbooks

### CODE REVIEW

File: runbooks/source/create-cluster.html.md.erb [hunk 1]
```diff
@@ -55,39 +55,39 @@ By default, the script will create a 'small' cluster. This means the master and
...
 variable "cluster_node_count_c" {
   description = "The number of worker node in the cluster in Availability Zone eu-west-2c"
   default = {
-    live-1  = "9"
+    live  = "9"
     default = "1"
   }
 }
 
 variable "master_node_machine_type" {
```
Comment (razvan-moj) [inline runbooks/source/create-cluster.html.md.erb:79]: we don't have masters anymore

File: runbooks/source/create-cluster.html.md.erb [hunk 2]
```diff
@@ -170,7 +170,7 @@ KubeDNS is running at https://api.XXXXXXX.cloud-platform.service.justice.gov.uk/
 
 ...where `XXXXXXX` is the name of your test cluster.
 
-If your context is pointing at live-1, switch to your test cluster by running
+If your context is pointing at live, switch to your test cluster by running
 
 ```bash
```
Comment (razvan-moj) [inline runbooks/source/create-cluster.html.md.erb:175]: no more Kops either

Comment (razvan-moj) [reply runbooks/source/create-cluster.html.md.erb:175]: aws eks update-kubeconfig

File: runbooks/source/manually-apply-namespace.html.md.erb [hunk 3]
```diff
@@ -30,20 +30,20 @@ export PINGDOM_API_KEY="XXXXXXXXXXXXX"
 ```
 
 
-## Target the live-1 cluster
+## Target the live cluster
 
 ```
 export AWS_PROFILE=moj-cp
 export KOPS_STATE_STORE=s3://cloud-platform-kops-state
```
Comment (razvan-moj) [inline runbooks/source/manually-apply-namespace.html.md.erb:37]: again, no more kops

File: runbooks/source/manually-apply-namespace.html.md.erb [hunk 4]
```diff
@@ -55,15 +55,15 @@ I'm using `offender-events-dev` for this example. Use whatever is relevant for y
...
-cd namespaces/live-1.cloud-platform.service.justice.gov.uk/${NAMESPACE}/resources
+cd namespaces/live.cloud-platform.service.justice.gov.uk/${NAMESPACE}/resources
 ```
 
 ## Terraform Init
 
 ```
 terraform init \
   -backend-config="bucket=cloud-platform-terraform-state" \
-  -backend-config="key=cloud-platform-environments/live-1.cloud-platform.service.justice.gov.uk/${NAMESPACE}/terraform.tfstate" \
```
Comment (razvan-moj) [inline runbooks/source/manually-apply-namespace.html.md.erb:66]: backend path is still live-1, this is important!

File: runbooks/source/manually-delete-namespace-resources.html.md.erb [hunk 5]
```diff
@@ -37,9 +37,9 @@ That's the code block [here](https://github.com/ministryofjustice/cloud-platform
 
 > If the script fails partway through, it may leave a new file in your local working copy:
 >
-> `namespaces/live-1.cloud-platform.service.justice.gov.uk/[namespace name]/resources/main.tf`
```
Comment (razvan-moj) [inline runbooks/source/manually-delete-namespace-resources.html.md.erb:40]: same here, path ion bucket is live-1

File: runbooks/source/tips-and-tricks.html.md.erb [hunk 6]
```diff
@@ -167,7 +167,7 @@ kubectl -n concourse-main create job ds-update-orphaned-statefiles --from=cronjo
 ## Hijack a concourse job container
 
 ```
-fly -t manager hijack -j environments-terraform/apply-namespace-changes-live-1
+fly -t manager hijack -j environments-terraform/apply-namespace-changes-live
```
Comment (razvan-moj) [inline runbooks/source/tips-and-tricks.html.md.erb:170]: environments-live

**Review by razvan-moj — CHANGES_REQUESTED** [review]

File: runbooks/source/create-cluster.html.md.erb (hunk 1 above)
Comment (sablumiah) [reply runbooks/source/create-cluster.html.md.erb:79]: I have cerated a separate ticket up update this doc as its the update script doc that is for kops and EKS

File: runbooks/source/manually-apply-namespace.html.md.erb (hunk 3 above)
Comment (sablumiah) [reply runbooks/source/manually-apply-namespace.html.md.erb:37]: I have cerated a separate ticket up update this doc as its the update script doc that is for kops and EKS

File: runbooks/source/manually-apply-namespace.html.md.erb (hunk 4 above)
Comment (sablumiah) [reply runbooks/source/manually-apply-namespace.html.md.erb]: changes reverted and separate ticket created

**Review by razvan-moj — APPROVED** [review]

**Pull request merged**